src/precice/impl/ParticipantState.cpp

```diff
@@ -360,6 +360,10 @@ void ParticipantState::exportInitial()
     PRECICE_DEBUG("Exporting initial mesh {} to location \"{}\"", context.meshName, context.location);
     context.exporter->doExport(0, 0.0);
   }
+
+  for (const PtrWatchPoint &watchPoint : watchPoints()) {
+    watchPoint->exportPointData(0.0);
```

Comment:
We should do the same for the WatchIntegrals for consistency

Reply:
Ok. Then I think I will also have to update the test accordingly. Will take care of it.

Reply:
Done.